### PR TITLE
initdb using migrations

### PR DESCRIPTION
**Pull request by golgher**
dispatchShell isn't working well and I was forced to change argv to initdb work.

### CODE REVIEW

**Comment by josegonzalez** [comment]
What do you mean by "isn't working well"?

**Comment by golgher** [comment]
@josegonzalez 
When I tried use dispatchShell without change argv I get this Exception:

[InvalidArgumentException]
Command "initdb" is not defined.

File: src/Shell/AclShell.php [hunk 1]
```diff
@@ -367,7 +367,10 @@ public function view()
      */
     public function initdb()
     {
-        return $this->dispatchShell('schema create DbAcl');
+        $_SERVER['argv'][1]='migrations';
+        $_SERVER['argv'][2]='migrate';
+        $_SERVER['argv'][3]='--plugin=acl';
```
Comment (markstory) [inline src/Shell/AclShell.php]: I wonder if this is a quirk of migrations shell being implemented mostly as extensions to Phinx.

Comment (lorenzo) [inline src/Shell/AclShell.php]: It sure is... Do we need this initdb method? Can't we just document that you need to run the migrations shell for this plugin?

**Comment by rchavik** [comment]
Merged without initdb changes.

Thanks

**Pull request closed without merging**